styleguide.config.styles.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
Looks like we didn't migrate sidebar background color, is this intentional?

Reply:
Oh! Good catch. Let me see if it's broken with / without it.

Reply:
I think we're good. 

![image](https://user-images.githubusercontent.com/841084/98313094-f1a37700-1f87-11eb-9060-64b5d8a21a77.png)

We used to have a background color because the old way of having a sticky header + left nav was some fixed positioning. When scrolling, it's transparent background was bleeding other background. But it seems fine from the screenshot now that we use a CSS grid to manage our top-level page sections.